Sming/SmingCore/Network/TcpServer.cpp

```diff
@@ -128,7 +128,7 @@ err_t TcpServer::onAccept(tcp_pcb *clientTcp, err_t err)
 void TcpServer::onClient(TcpClient *client)
 {
 	activeClients++;
-	debugf("TcpServer onClient  %s, activeClients = %d\r\n ",client->getRemoteIp().toString().c_str(),activeClients);
+	debugf("TcpServer onClient  %s\r\n ",client->getRemoteIp().toString().c_str());
```

Comment:
Sharp.. Should I update the PR or submit a new one ?

PS will be offline from now. Further tomorrow